crates/web-sys/webidls/enabled/WebAuthentication.webidl

```diff
@@ -21,6 +21,11 @@ partial interface PublicKeyCredential {
     static Promise<boolean> isUserVerifyingPlatformAuthenticatorAvailable();
 };
 
+[SecureContext]
+partial interface PublicKeyCredential {
+    static Promise<boolean> isConditionalMediationAvailable();
+};
```

Comment:
This has to go into the unstable folder.

Reply:
@Firstyear the way to move it to the unstable folder is to actually move the WebIDL to the [unstable folder](https://github.com/rustwasm/wasm-bindgen/tree/165ad00ccff4bc67705afdacecf894cb8250ee2e/crates/web-sys/webidls/unstable).